lib/omniauth/strategies/saml.rb

```diff
@@ -135,7 +136,7 @@ def other_phase
         Hash[found_attributes]
       end
 
-      extra { { :raw_info => @attributes } }
+      extra { { :raw_info => @attributes, :response_object =>  @response_object} }
```

Comment:
Nitpick: can you add the missing space before the closing brace (after `@response_object`)